src/components/Header.js

```diff
@@ -22,7 +22,11 @@ function Header() {
           <a href="/">
             <img src={logo} alt={logo} />
           </a>
-          {user ? userInformation({ user }) : <button>Login</button>}
+          {user ? (
+            userInformation({ user })
+          ) : (
+            <button className="headerButton">Login</button>
+          )}
         </div>
       </div>
     </header>

```

Comment:
`header` -> `div class: nav-space` -> `div class: nav`로 마크업이 되어 있는데  div를 하나 줄여도 문제가 되지 않아 보이거든요. 마크업 하실때 블록요소와 인라인요소를 잘 구분하셔서 같은 UI라면  최대한 마크업을 간결하게 해주세요.
예를 들면 아래와 같이 div를 하나 없애고 블록요소인 header에 클래이름을 주어 스타일링을 하고 클래스 이름을 `nav`로 주는게 아닌 nav`태그를 써줌으로써 좀 더 명확하게 마크업을 하는거죠

```suggestion
<header className="nav-space">
            <nav className="nav">
                <a href="/">
                    <img src={logo} alt={logo} />
                </a>
                {user ? userInformation({ user }) : <button className="headerButton">Login</button>}
            </nav>
        </header>

```